modules/admin/proc-install-operator.adoc

```diff
@@ -19,23 +23,38 @@ Containers are available for the following CPU architectures:
 * You are logged in as an administrator on the {ocp-short} web console.
 * You have configured the appropriate roles and permissions within your project to create an application. For more information, see the link:https://docs.openshift.com/container-platform/{ocp-version}/applications/index.html[Red Hat OpenShift documentation on Building applications].
 
-.Procedure
+[NOTE]
+====
+For enhanced security, deploy the {product} Operator in a dedicated default namespace such as `rhdh-operator`. The cluster administrator can restrict other users' access to the Operator resources through role bindings or cluster role bindings.
+====
 
-. In the *Administrator* perspective of the {ocp-short} web console, navigate to *Operators* > *OperatorHub*.
+.Procedure
 
+. In the *Administrator* perspective of the {ocp-short} web console, navigate to *Operators* -> *OperatorHub*.
 . In the *Filter by keyword* box, enter {product-short} and click the *{product}* Operator card.
-
 . On the *{product} Operator* page, click *Install*.
 
-. On the *Install Operator* page, click *Install*.
+. On the *Install Operator* page, use the *Update channel* drop-down menu to choose an update channel:
```

Comment:
```suggestion
. On the *Install Operator* page, use the *Update channel* drop-down menu to select the update channel that you want to use:
```
_click_ might also work here, but IBM advises against the word _choose_ in reference to UI actions. 
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#c